cluster/juju/layers/kubernetes-worker/actions.yaml

```diff
@@ -23,17 +23,17 @@ registry:
         description: An htpasswd file used for authentication.
       tlscert:
         type: string
-        description: base64 encoded TLS certificate for the registry.
+        description: TLS certificate for the registry.
```

Comment:
+1 to not making the user bae64 encode the values I think this will be a better user experience. Even though its only a single removal of a piped command. 👍